packages/components/src/components/text-input/_text-input.scss

```
@@ -64,6 +64,28 @@
     background-color: $field-02;
   }
 
+  //-----------------------------
+  // Fluid Text Input
+  //-----------------------------
+  .#{$prefix}--text-input-wrapper--fluid {
+    position: relative;
+  }
+  .#{$prefix}--text-input-wrapper--fluid .#{$prefix}--label {
+    position: absolute;
+    top: rem(14px);
+    left: rem(14px);
+    z-index: 1;
+  }
+
+  .#{$prefix}--text-input-wrapper--fluid .#{$prefix}--form__helper-text {
+    display: none;
+  }
+
+  .#{$prefix}--text-input-wrapper--fluid .#{$prefix}--text-input {
+    min-height: rem(64px);
+    padding: rem(32px) rem(14px) $carbon--spacing-05;
```

I believe we want `16px` of padding on the sides

```suggestion
    padding: rem(32px) $carbon--spacing-05 $carbon--spacing-05;
```